en/04_Changelogs/4.12.0.md

```diff
@@ -267,14 +270,20 @@ Silverstripe CMS 4 includes multiple tasks to migrate files to newer states. The
 
 - `extra_fields` data for a [ManyManyList](api:SilverStripe\ORM\ManyManyList) can now be set easily with the ORM - simply call the [setExtraData()](api:SilverStripe\ORM\ManyManyList::setExtraData()) method. See [the `many_many` documentation](/developer_guides/model/relations#many-many) for more details.
 
-## Bugfixes {#bugfixes}
+## Deprecating APIs ahead of Silverstripe CMS 5 {#deprecation-cms5}
 
-This release includes a number of bug fixes to improve a broad range of areas. Check the change logs for full details of these fixes split by module. Thank you to the community members that helped contribute these fixes as part of the release!
+Substantial work went into normalising our deprecation warnings and identifying code that will be removed in Silverstripe CMS 5. This will make it easier for Silverstripe CMS 4 projects to upgrade to CMS 5 by clearing marking which APIs will be removed in the next major release. Deprecated APIs are still part of the official Silverstripe CMS 4 codebase and can be still be used. Deprecated code is marked with the `@deprecated` PHPDoc annotation.
```

Comment:
```suggestion
Substantial work went into normalising our deprecation warnings and identifying code that will be removed in Silverstripe CMS 5. This will make it easier for Silverstripe CMS 4 projects to upgrade to CMS 5 by clearly marking which APIs will be removed in the next major release. Deprecated APIs are still part of the official Silverstripe CMS 4 codebase and can still be be used. Deprecated code is marked with the `@deprecated` PHPDoc annotation.
```